modules/local/gtdbtk_classify.nf

```diff
@@ -47,7 +47,7 @@ process GTDBTK_CLASSIFY {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        gtdbtk: \$(gtdbtk --version | sed "s/gtdbtk: version //; s/ Copyright.*//")
+        gtdbtk: \$(gtdbtk --version | head -n 1 | sed "s/gtdbtk: version //; s/ Copyright.*//")
```

Comment:
I have bad experience with `head` in that case, see https://github.com/nf-core/mag/pull/236, so I propose
```suggestion
        gtdbtk: \$(gtdbtk --version | sed -n 1p | sed "s/gtdbtk: version //; s/ Copyright.*//")
```